include/RAJA/pattern/params/reducer.hpp

```diff
@@ -83,16 +55,41 @@ namespace detail
   // Basic Reducer
   //
   //
-  template <typename Op, typename T>
+
+  // Basic data type Reducer
+  // T must be a basic data type
+  // VType must be ValOp<T, Op>
+  template <typename Op, typename T, typename VType>
```

Comment:
Would it be more clear to `VType` as `TOp` or something else with op to better indicate that its a ValOp type?

Reply:
Changing this to `VOp`.